src/frontend/src/pages/PIAIntakeForm/index.tsx

```diff
@@ -474,7 +476,22 @@ const PIAFormPage = () => {
 
     return () => clearTimeout(autoSaveTimer);
   });
-
+  const handleBack = () => {
+    navigate(-1);
```

Comment:
is "back" based on the previous tab or previous page?

Reply:
previous tab is my understanding cc @kushal-arora-fw   double check

Reply:
That is correct, previous tab it is.